content/en/storage-providers/advanced-configurations/workers.md

```diff
@@ -0,0 +1,685 @@
+
+- Number of CPU threads the task will use.
+- Minimum amount of RAM required for good performance.
+- Maximum amount of RAM required to run the task, where the system can swap out part of the RAM to disk, and the performance won't be affected too much.
+- Whether the system has a GPU.
+
+### Task resource table
+
+The default resource table lives in [resources.go](https://github.com/filecoin-project/lotus/blob/master/extern/sector-storage/storiface/resources.go) and can be edited to tune the scheduled behavior to fit specific sealing clusters better.
+
+Default resource value table. Some of these values are _fairly_ conservative:
+
+| Sector size | Task Type  | Threads | Min RAM | Min disk space| GPU        |
+|-------------|------------|---------|---------|------------|------------|
+|     32G     | AddPiece   | 1*      | 4G      | 4G         |            |
+|             | PreCommit1 | 1**     | 56G     | 64G        |            |
+|             | PreCommit2 | 92%***  | 15G     | 15G        | If Present |
+|             | Commit1    | 0****   | 1G      | 1G         |            |
+|             | Commit2    | 92%***  | 32G+30G | 32G+150G   | If Present |
+|     64G     | AddPiece   | 1*      | 8G      | 8G         |            |
+|             | PreCommit1 | 1**     | 112G    | 128G       |            |
+|             | PreCommit2 | 92%***  | 30G     | 30G        | If Present |
+|             | Commit1    | 0****   | 1G      | 1G         |            |
+|             | Commit2    | 92%***  | 64G+60G | 64G+190G   | If Present |
+
+\* AddPiece can use multiple threads, it's likely that this value will change in near future\
+** When used with the `FIL_PROOFS_USE_MULTICORE_SDR=1` env var, PreCommit1 can use multiple cores (up to the number of cores sharing L3 caches)\
+*** Depending on the number of available threads, this value means:
+
+```
+ 12  * 0.92 = 11
+ 16  * 0.92 = 14
+ 24  * 0.92 = 22
+ 32  * 0.92 = 29
+ 64  * 0.92 = 58
+ 128 * 0.92 = 117
+```
+
+**** The Commit1 step is very cheap in terms of CPU time and blocks the Commit2 step. Allocating it to zero threads makes it more likely it will be scheduled with higher priority.
+
+The Unseal task has the same resource use as the PreCommit1 task.
+
+{{< alert icon="info" >}}
+The default and custom configurations in the task resource table can be overridden using environment variables on a per worker basis. See the [environment variables]({{< relref "#control-groups" >}}) section for information. You can also gather these details using the `lotus-worker resources --default` command.
+{{< /alert >}}
+
+### Resource windows
+
+The scheduler uses the concept of resource windows to prevent resource starvation of tasks requiring larger amounts of resources by tasks with smaller resource requirements.
```

Comment:
```suggestion
The scheduler uses the concept of resource windows to prevent _resource starvation_ of tasks requiring larger amounts of resources by tasks with smaller resource requirements.
```